main.py

```
@@ -2,16 +2,15 @@
 import sys
 sys.dont_write_bytecode = True
 from utils.operator.cleanup import cleanup
-from cli.args import parse_args
-from cli.commands import install_operator, install_operators
+from cli.args import parse_args, build_config, select_operators
```

Ruff (0.11.9)

`cli.args.select_operators` imported but unused

Remove unused import: `cli.args.select_operators`

(F401)